main.py

```diff
@@ -1,7 +1,47 @@
-import init_django_orm  # noqa: F401
+import init_django_orm  # noqa F401
 
 from django.db.models import QuerySet
 
 
+from db.models import Genre, Actor
+
+
 def main() -> QuerySet:
     pass
+    genres_data = [("Western",), ("Action",), ("Drama",)]
+
+    for genre_name, in genres_data:
```

Comment:
There is no need to create a list of tuples since they contain only one string. Create a list of strings instead and iterate through them